apps/src/codebridge/FileBrowser/styles/filebrowser.module.scss

```diff
@@ -4,13 +4,25 @@
   height: 100%;
   background-color: $light_gray_950;
   color: $white;
+  overflow: hidden;
 }
 
 .fileBrowserHeader {
   background-color: $light_gray_950;
   border-bottom: 1px solid #54595E;
 }
 
+.fileBrowserContents {
+  overflow-y: auto;
+  scrollbar-color: $light_gray_500 $light_gray_950;
+}
+
+.fileBrowserContents::-webkit-scrollbar {
```

Comment:
I'm not sure if this style rule is applying? I tried a different color here and the color did not change.
When I tried different colors for the thumb and track above (line 17), the colors did change but not consistently across the 4 browsers.
I had similar issues when implementing slider for music lab play view https://github.com/code-dot-org/code-dot-org/pull/58293 - ended up creating a component bc the slider styling got so complicated https://github.com/code-dot-org/code-dot-org/pull/58373.
I think this is definitely an improvement! But maybe if there's time later on, we can improve on styling?

Reply:
oh yep I meant to delete this and forgot, it doesn't work. I noted in the description that I was just going to go with our default styling for safari since it looks fine 😄

Reply:
deleted!